libs/ha/ha_common_libs_k8s.py

```diff
@@ -1204,46 +1205,42 @@ def setup_mock_monitor(node_obj):
         :param node_obj: Object for node
         :return: Bool
         """
-        # TODO: Revisit once RPM for mock_health_event_publisher is available
         LOGGER.info("Get HA pod name for setting up mock monitor")
         ha_pod = node_obj.get_all_pods(pod_prefix=common_const.HA_POD_NAME_PREFIX)[0]
-        LOGGER.info("Install git inside ha pod %s", ha_pod)
+        LOGGER.info("Copy script %s inside %s", common_const.MOCK_MONITOR_LOCAL_PATH, ha_pod)
         try:
-            node_obj.send_k8s_cmd(operation="exec", pod=ha_pod, namespace=common_const.NAMESPACE,
-                                  command_suffix="yum install git -y", decode=True)
-        except IOError as error:
-            LOGGER.error("Failed to install git inside ha pod %s due to error: %s", ha_pod, error)
+            LOGGER.info("Copying file %s to %s", common_const.MOCK_MONITOR_LOCAL_PATH,
+                        common_const.MOCK_MONITOR_REMOTE_PATH)
+            node_obj.copy_file_to_remote(local_path=common_const.MOCK_MONITOR_LOCAL_PATH,
```

Comment:
This function already except the excation. Why again handling the same insted of checking return value. ?